pkg/project/resource_project_repo.go

```diff
@@ -127,10 +127,15 @@ var updateRepos = func(projectKey string, terraformRepoKeys []RepoKey, m interfa
 	return readRepos(projectKey, m)
 }
 
+var retry5xxRange = func(response *resty.Response, err error) bool {
+	return response.StatusCode() >= 500 && response.StatusCode() <= 599
```

Comment:
This convo isn't resolved. Not all 500 errors should be retried.

Reply:
We should check `response.IsError()` and the response body.

Reply:
retrying only on 408, 500, 502, 503, 504. Code updated